code/modules/guncrafting/ammo.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/ammo_casing/energy/prototype
+	name = "prototype energyn lens"
```

Comment:
It was clearly an attempt at creating a meme in the style of energy eall. Sadly this time it got noticed before it could make its way into view of the players